ospfd/ospf_vty.c

```diff
@@ -9558,7 +9558,7 @@ DEFUN (ospf_default_information_originate,
 
 DEFUN (no_ospf_default_information_originate,
        no_ospf_default_information_originate_cmd,
-       "no default-information originate [{always|metric (0-16777214)|metric-type (1-2)|route-map RMAP_NAME}]",
+       "no default-information originate [{always|metric [(0-16777214)]|metric-type [(1-2)]|route-map RMAP_NAME}]",
```

Comment:
Can't the route-map name also be optional for the "no" form?

Reply:
OK, I will fix it